packages/mobile/src/screens/AxiosExample.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import { View, StyleSheet, ScrollView } from 'react-native';
+import News from '../components/News';
+import axios from 'axios';
+
+class AxiosExample extends Component{
+  constructor(props){
+		super(props);
+		this.state = {
+			news: []
+		}
+	}
+	componentDidMount() {
+    axios.get('https://newsapi.org/v2/everything?q=netflix&sortBy=publishedAt&apiKey=c04fd04b98d841d9ac3f0826f2497094')
```

Comment:
skoro ustawiłeś baseurl to czemu tu podajesz znów cału url?
(wiem że to tylko przykład ale informacyjnie: komponent nie powinien samodzielnie wykonywać requestów -> powinien to robić przez akcje a akcja powinna wywoływać connector/serwis i w serwis powinien sie znaleść dopiero axios.get() - to tylko informacynie tego z nawiasu nie musisz implemetować)